contracts/message-bus/MessageInbox.sol

```diff
@@ -15,7 +15,7 @@ pragma solidity >=0.5.0 <0.6.0;
 // limitations under the License.
 
 import "./MessageBox.sol";
-import "./StateRootI.sol";
+import "./StateRootInterface.sol";
```

Comment:
As we have touched this section, i think we must sort the imports.

Reply:
Done.